build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -14,6 +14,7 @@ public class Crossgen
         private string _jitVersion;
         private string _crossGenPath;
         private string _targetRID;
+        private static readonly string s_transportPackagePrefix = "transport";
```

Comment:
I don't think we expect to, but if we have the need to, this simplifies it ever so slightly.